pkg/machines/libvirt-dbus.js

```diff
@@ -644,7 +644,7 @@ LIBVIRT_DBUS_PROVIDER = {
 
     GET_NODE_MAX_MEMORY({ connectionName }) {
         if (connectionName) {
-            return dispatch => call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'NodeGetMemoryStats', [0, 0], TIMEOUT)
+            return dispatch => call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'NodeGetMemoryStats', [-1, 0], TIMEOUT)
```

Comment:
Do you mind adding a comment here that explains the -1? Thanks!

Reply:
According to https://libvirt.org/html/libvirt-libvirt-host.html -1 will return the memory in all cells, except of just the cell 0. Some systems have many cells and returning the memory from just one is unreliable.

Reply:
Right, I know -- Sorry, I meant to change your commit to add a comment into the *code*. Something like

    # -1 == VIR_NODE_MEMORY_STATS_ALL_CELLS

Reply:
Comment added